docs/guides/campaign_recipes/Open-Source-Loyalty-Accelerator.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Open Source Loyalty Accelerator for Omnichannel Retail
```

Comment:
Sprawdź proszę, jak to wygląda na stronce, bo ten tytuł jest bardzo długi. Jeśli się hardkorowo zawinie na 2-3 linijki, to go skróć.